File: projects/m1/006-sum-of-digits-in-a-integer/solutions/GABRIELE DIANA/m1_006.py
# Develop a program that reads a four-digit integer from the user and displays the sum of its digits. 
# For example, if the user enters 3141 then your program should display 3+1+4+1=9.

numero = input("inserisci 4 cifre : ")

M = int(numero[0])
C = int(numero [1])
D = int(numero [2])
U = int(numero [3])

print(str(M) + "+" + str(C) + "+" + str(D) + "+" + str(U) + "=" + str(M+C+D+U))

File: projects/m1/006-sum-of-digits-in-a-integer/solutions/dariogambarro/m1-006.js
let num = +prompt('Digit a four-digit number');
let d1;
let d2;
let d3;
let d4;
let sum;
let mod;

mod = num % 1000;
d1 = (num - mod) / 1000;

num = num - (d1 * 1000);
mod = num % 100;
d2 = (num - mod) / 100;

num = num - (d2 * 100);
mod = num % 10;
d3 = (num - mod) / 10;

d4 = num - (d3 * 10);

sum = d1 + d2 + d3 + d4;

console.log(`the sum of digits is ${sum}`);